src/main/java/fiji/plugin/volumeviewer/Volume_Viewer.java

```diff
@@ -118,6 +118,7 @@ public void run(String args) {
 		}
 		
 		imp = WindowManager.getCurrentImage();
+		String stackName = imp.getTitle();
```

Comment:
This should go after the `null` check for `imp`, right? I suggest to move it right in front of the `JFrame` creation.

Reply:
You are correct, fixed